jetbrains-gateway/README.md

```diff
@@ -14,7 +14,7 @@ This module adds a JetBrains Gateway Button to open any workspace with a single
 ```tf
 module "jetbrains_gateway" {
   source         = "registry.coder.com/modules/jetbrains-gateway/coder"
-  version        = "1.0.23"
+  version        = "1.0.25"
```

Comment:
Since we are changing the default behavior, do we want to do a more aggressive version bump?

Reply:
Yes. We can do that but that will make all future versions of other modules go to 1.1.X from now on. That would cause confusion for the users of other modules. 

I can also remove the change in default behaviour in this PR and do once we have #157